### PR TITLE
User guide: Ensure user name must be 'ubuntu'

### PR DESCRIPTION
**Pull request by hiltonlima**
- Removed some TH image mentions
- Add TH support to Raspberry PI 5
- Ensure the host user name must be 'ubuntu'

Script PR: https://github.com/project-chip/certification-tool/pull/346

### CODE REVIEW

**Review by gladystonfranca — APPROVED** [review]

**Review by rquidute — APPROVED** [review]

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 1]
```diff
@@ -104,6 +104,8 @@ endif::[]
                                                                      * Fix the Thread RCP Firmware link. +
                                                                      * Add Nrfconnect Sample APPs Firmwares section with link.
 | 36          | 09-Aug-2024  | [Apple] Antonio Melo Jr.            | * Adding the new cleanup procedure to the troubleshooting of TH Installation section.
+| 37          | 13-ug-2024   | [Apple] Hilton Lima                 | * Remove old references for TH image (obsolete).
```
Comment (antonio-amjr) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:107]:
```suggestion
| 37          | 13-Aug-2024   | [Apple] Hilton Lima                 | * Remove old references of TH image (obsolete).
```

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 2]
```diff
@@ -104,6 +104,8 @@ endif::[]
...
+                                                                     * Add a warning to ensure the username needs to be 'ubuntu'.
```
Comment (antonio-amjr) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:108]:
```suggestion
                                                                     * Add a warning to ensure that the username needs to be 'ubuntu'.
```

**Review by antonio-amjr — CHANGES_REQUESTED** [review]

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 3]
```diff
@@ -532,19 +537,19 @@ A hobby developer can build Matter reference apps either using a Raspberry Pi or
...
 * Go to the apps folder in /home/ubuntu/apps (as shown below) and launch the app that the user is interested in.
 
 image:images/img_3.png[]
 
 To build the example apps from the latest SDK source, follow the instructions below: 
 
-* User to acquire Raspberry Pi Version 4 with SD card of minimum 64 Gb memory.
-* Flash the TH image on to the SDK card that will be inserted into the Raspberry Pi as the TH image comes with the default Ubuntu OS image OR the user can download the latest Ubuntu LTS image and install all the required dependencies as outlined in https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md[https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md].
+* User to acquire Raspberry Pi Version (4 or 5) with SD card of minimum 64 GB memory.
+* Flash the latest Ubuntu LTS image and install all the required dependencies as outlined in https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md[https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md].
```
Comment (antonio-amjr) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:552]:
```suggestion
* Do a fresh install of the {ubuntu-description} image and install all the required dependencies as outlined in https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md[https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md].
```

**Review by antonio-amjr — CHANGES_REQUESTED** [review]

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 4]
```diff
@@ -104,6 +104,8 @@ endif::[]
                                                                      * Fix the Thread RCP Firmware link. +
                                                                      * Add Nrfconnect Sample APPs Firmwares section with link.
 | 36          | 09-Aug-2024  | [Apple] Antonio Melo Jr.            | * Adding the new cleanup procedure to the troubleshooting of TH Installation section.
+| 37          | 13-Aug-2024   | [Apple] Hilton Lima                 | * Remove old references of TH image (obsolete).
```
Comment (antonio-amjr) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:107]:
```suggestion
| 37          | 13-Aug-2024   | [Apple] Hilton Lima                | * Remove old references of TH image (obsolete).
```

**Comment by antonio-amjr** [comment]
It seems that the `warning` affected the structure of the [4.1.2. TH Installation on Raspberry Pi](https://github.com/project-chip/certification-tool/pull/366#th-installation-on-raspberry-pi) steps **numbering**:

**Before:**
Screenshot 2024-08-14 at 14 27 14

**After:**
Screenshot 2024-08-14 at 14 27 23

**Review by antonio-amjr — CHANGES_REQUESTED** [review]
Please, try to solve the steps structure of the above comment.

**Review by antonio-amjr — APPROVED** [review]

**Pull request merged**